frontend/src/Pages/AdminPage.js

```diff
@@ -44,18 +64,23 @@ export default function AdminPage() {
     return (
       <Page>
         <Loading />
+        <BottomNav />
       </Page>
     )
   }
 
   return (
-    <MainPage>
+    <PageHeader>
```

Comment:
The whole page is wrapped by a header? 😕

Reply:
No :D Got a page without header (Page) and one with header (PageHeader). Gonna change the name to PageWithHeader 👍